src/components/datepicker/localeUtils.ts

```diff
@@ -1,7 +1,7 @@
 import { DateTime, Info } from 'luxon';
 
 const defaultLocale = 'en-GB';
-const firstDayOfWeek = {
+const firstDayOfWeek: Record<string, number> = {
```

Comment:
PP: use `as const` instead

```tsx
const firstDayOfWeek = {
 'da-DK': 1,
  'de-DE': 1,
  'de-DE': 1,
  'en-GB': 1,
  'en-GB': 1,
  'en-US': 0,
  'en-US': 0,
  'es-ES': 1,
  'es-ES': 1,
  'fi-FI': 1,
  'fi-FI': 1,
  'fr-BE': 1,
  'fr-BE': 1,
  'fr-FR': 1,
  'fr-FR': 1,
  'it-IT': 1,
  'it-IT': 1,
  'nb-NO': 1,
  'nb-NO': 1,
  'nl-BE': 1,
  'nl-BE': 1,
  'nl-NL': 1,
  'nl-NL': 1,
  'pl-PL': 1,
  'pl-PL': 1,
  'pt-PT': 1,
  'pt-PT': 1,
  'sv-SE': 1,
  'sv-SE': 1,
  'tr-TR': 1,
  'tr-TR': 1,
} as const;
```

Reply:
I can't find a way to fix this when using `as const`

![image](https://user-images.githubusercontent.com/55881661/168978482-f646fbc4-825b-49b8-99e9-69f010ad3251.png)

Reply:
locale will have to be typed as `keyof firstDayOfWeek` then 🙈 Probably best to put all the locales in a different type